src/core/operations/DateTime.js

```diff
@@ -79,6 +81,41 @@ const DateTime = {
 
 
     /**
+     * Converts a Windows FILETIME to Unix Epoch time.
+     *
+     * @param {string} input
+     * @param {Object[]} args (not used)
+     * @returns {string}
+     */
+    runFromFiletimeToUnix: function(input, args) {
+        input = new Decimal(input);
+        input = input.sub(116444736000000000).div(10000000);
+        if (input.gte(0) && input.lt(Math.pow(2, 31))){
+            return input.toString();
+        } else {
+            throw "Date " + input + " is not a valid date";
```

Comment:
Perhaps make this a little more descriptive. How about `"The value " + input + " cannot be expressed as a UNIX timestamp."`?